package.json

```diff
@@ -2,22 +2,25 @@
   "name": "cozy-notes",
   "version": "1.17.0",
   "scripts": {
-    "tx": "tx pull --all || true",
-    "lint": "yarn lint:js && yarn lint:styles",
-    "lint:js": "cs lint '{src,test}/**/*.{js,jsx}'",
-    "lint:styles": "stylint src/styles --config ./.stylintrc",
-    "prebuild": "yarn lint",
-    "build": "yarn build:browser",
     "build:browser:dev": "cs build --browser --development",
     "build:browser": "cs build --browser",
     "build:mobile": "cs build --mobile",
-    "watch": "yarn watch:browser",
-    "watch:browser": "cs watch --browser",
-    "watch:mobile": "cs watch --mobile",
-    "start": "cs start --browser",
+    "build": "yarn build:browser",
+    "cozyPublish": "cs publish --token $REGISTRY_TOKEN --build-commit $(git rev-parse ${DEPLOY_BRANCH:-build})",
     "deploy": "git-directory-deploy --directory build/ --branch ${DEPLOY_BRANCH:-build} --repo ${DEPLOY_REPOSITORY:-https://cozy-bot:$GITHUB_TOKEN@github.com/cozy/cozy-notes}",
+    "install:clean:start": "yarn install:clean && yarn start",
+    "install:clean": "rm -rf node_modules && rm -f yarn.lock && yarn install",
+    "lint:js": "cs lint '{src,test}/**/*.{js,jsx,ts,tsx}'",
+    "lint:styles": "stylint src/styles --config ./.stylintrc",
+    "lint": "yarn lint:js && yarn lint:styles",
+    "prebuild": "yarn lint",
+    "stack:docker:dev": "COZY_STACK_HOST=cozy.tools docker run --rm -it -p 8080:8080 -p 5984:5984 -e COZY_DISABLE_CSP=1 -v $(pwd)/build:/data/cozy-app/app cozy/cozy-app-dev",
```

Comment:
We are trying to move from `cozy.tools` to `cozy.localhost`, is it possible to do it here?

Reply:
Updated